### PR TITLE
test/cql-pytest - aggregation tests

### PR DESCRIPTION
**Pull request by nyh**
This small series reorganizes the existing functional tests for aggregation (min, max, count) and adds additional tests for sum reproducing the strange (but Cassandra-compatible) behavior described in issue #13027.

### CODE REVIEW

File: test/cql-pytest/test_aggregate.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
-# Copyright 2023-present ScyllaDB
+# Copyright 2021-present ScyllaDB
```
Comment (denesb) [inline test/cql-pytest/test_aggregate.py:1]: Why was copyright changed?

Comment (denesb) [reply test/cql-pytest/test_aggregate.py:1]: Or maybe this is just git generating strange diffs?

**Review by denesb — APPROVED** [review]

File: test/cql-pytest/test_aggregate.py (hunk 1 above)
Comment (nyh) [reply test/cql-pytest/test_aggregate.py:1]: As I explained in the commit message, I combined two test files, one supposedly about min() and max() and the second about count(), into one test file about all the aggregations (the above and also sum()). The copyright I used for the new file was the earlier of the two. Git decided arbitrarily to consider this patch a "rename" of one of the files, which explains what the patch looks like.

Comment (denesb) [reply test/cql-pytest/test_aggregate.py:1]: Right, I thought this was the case, git does generate counter-intuitive diffs sometimes.

**Comment by denesb** [comment]
Queued.

**Pull request merged**